src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
 public class Application {
+    static final int countOfBalls = 3;
```

Comment:
Add access modifier and follow naming conventions
```suggestion
    private static final int COUNT_OF_BALLS = 3;
```